### PR TITLE
Videopress: Add initial plugin skeleton

### PR DESCRIPTION
**Pull request by dhasilva**
#### Changes proposed in this Pull Request:

* Adds the initial structure for the Videopress plugin, generated with `jetpack generate`.

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Testing instructions:

**Build / Watch the plugin**

```
jetpack build plugins/videopress
```

```
jetpack watch plugins/videopress
```

Check you see the plugin in the plugins page. Link the plugin in your local testing site:

```
Jetpack docker link-plugin videopress
```

Plugin page:

Screen Shot 2022-07-29 at 6 22 38 PM

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202676535745533

### CODE REVIEW

**Comment by retrofox** [comment]
```cli
jetpack build plugins/videopress
```

image

File: projects/plugins/videopress/tests/e2e/package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build": "pnpm jetpack build plugins/videopress plugins/jetpack -v --no-pnpm-install --production",
+		"clean": "rm -rf output",
+		"config:decrypt": "openssl enc -md sha1 -aes-256-cbc -d -pass env:CONFIG_KEY -in ./node_modules/jetpack-e2e-commons/config/encrypted.enc -out ./config/local.cjs",
+		"distclean": "rm -rf node_modules",
+		"env:up": "e2e-env start --activate-plugins boost",
+		"env:down": "e2e-env stop",
+		"env:reset": "e2e-env reset --activate-plugins boost",
```
Comment (leogermani) [inline projects/plugins/videopress/tests/e2e/package.json:24]: we should remove this boos reference from the starter plugin

**Review by retrofox — APPROVED** [review]
LGTM 🚢 

image

**Comment by retrofox** [comment]
> One last step: head over to your WordPress.com diff, D85008-code, and deploy it.

r249908-wpcom

**Pull request merged**